src/core/src/main/resources/org/apache/jmeter/resources/messages.properties

```
@@ -1035,7 +1035,7 @@ sample_name_prefix=Prefix
 sample_name_suffix=Suffix
 sample_naming_scheme=Naming scheme
 sample_name_transaction=Transaction name
-sample_naming_format_help=<html>Format for the names of the samplers.<br> The counter, path and transaction names can be given with the placeholders <tt>#{counter}</tt>, <tt>#{path}</tt> and <tt>#{name}</tt>.</html>
+sample_naming_format_help=<html>Format for the names of the samplers.<br> The following placeholders can be used...<br> <tt>#{counter}</tt>, <tt>#{path}</tt>, <tt>#{name}</tt>, <tt>#{method}</tt>, <tt>#{host}</tt>, <tt>#{scheme}</tt>, <tt>#{port}</tt> and <tt>#{url}</tt></html>
```

I think, I would drop the ellipsis.
```suggestion
sample_naming_format_help=<html>Format for the names of the samplers.<br> The following placeholders can be used<br> <tt>#{counter}</tt>, <tt>#{path}</tt>, <tt>#{name}</tt>, <tt>#{method}</tt>, <tt>#{host}</tt>, <tt>#{scheme}</tt>, <tt>#{port}</tt> and <tt>#{url}</tt></html>
```